tests/fs-tree/fs-test.js

```diff
@@ -680,6 +680,88 @@ describe('FSTree fs abstraction', function() {
       });
     });
 
+
+    describe('.mkdirpSync', function() {
+      it('-> directory (create)', function() {
+        expect(tree.changes()).to.eql([]);
+        expect(tree.mkdirpSync('new-directory/a/b/c/')).to.eql(undefined);
+
+        let changes = tree.changes();
+
+
+        let operation = changes[3][0];
+        let relativePath = changes[3][1];
+        let entry = changes[3][2];
+
+        expect(operation).to.eql('mkdir');
+        expect(relativePath).to.eql('new-directory/a/b/c');
+        expect(entry).to.have.property('relativePath', 'new-directory/a/b/c');
+        expect(entry).to.have.property('mode');
+        expect(isDirectory(entry)).to.eql(true);
+        expect(entry).to.have.property('mtime');
+        expect(tree.changes()).to.have.property('length', 4);
```

Comment:
👍 we probably also want to assert that those changes are as we expect and that they're in the right order

ie check that they're
```js
[
  ['mkdir', 'new-directory', entryNewDir],
  ['mkdir', 'new-directory/a', entryA],
  ['mkdir', 'new-directory/a/b', entryB],
  ['mkdir', 'new-directory/a/b/c', entryC],
]
```